assets/scss/_custom.scss

```diff
@@ -0,0 +1,9 @@
+.td-cover--image {
+    background-image: url('../../../../images/hero-banner.jpg');
```

Comment:
as far as I know, this won't work. this should be set in the main repo config file